### DISCUSSION TITLE
Provide the CLI via package managers

### DISCUSSION THREAD

**Discussion by ph00lt0** [ente-io/ente#707] [Enhancements]
Make it it possible to install ente-cli via package managers, such as home brew.

**Comment by aliesbelik** [comment]
Manifest for [scoop](https://scoop.sh/) tested and available here:
https://github.com/aliesbelik/poldi/blob/main/bucket/ente-cli.json

**Comment by creedda** [comment]
would love to see the desktop app available via brew as well

**Reply by mnvr** [reply]
desktop app is already available via brew
```
brew info ente
```

**Reply by creedda** [reply]
found it. Thanks!

**Comment by phamleduy04** [comment]
I can't find ente-cli on brew. When I do `brew search ente` brew only return casks `ente` and `ente-auth`

**Reply by mathieutu** [reply]
I just made a [PR](https://github.com/Homebrew/homebrew-core/pull/201919) to Homebrew.

**Comment by mngshm** [comment]
I think this can also largely be a community effort to add ente apps and CLI to all the widely available package managers! 

I'll most probably start with adding ente desktop apps and CLI to AUR as I use arch daily.

**Reply by nk521** [reply]
For debian, one can rely on [dh-make-golang](https://people.debian.org/~stapelberg/2015/07/27/dh-make-golang.html). Maybe the team can use it to release deb packages on github.

**Comment by mathieutu** [comment]
I made a [PR](https://github.com/Homebrew/homebrew-core/pull/201919) to Homebrew.

**Reply by mathieutu** [reply]
It's been months we are waiting for the new release to be able to deploy it. 
@mngshm do you think you could tag it?

Thanks.

**Reply by mngshm** [reply]
I read the PR thread, what is it that needs to be done from Ente's side, I couldn't understand that conversation much. We haven't had any new changes to the CLI recently.

**Reply by mathieutu** [reply]
Basically, [this commit](https://github.com/ente-io/ente/commit/cfae8523a18bdc89ad7de3fb5368fbd4408adeae) is the one we need.
So a minor tag with the changes since the last release should do the job.

**Reply by ua741** [reply]
Sorry for the unnecessary delay. We will make a release early next week for cli.

**Reply by mathieutu** [reply]
Great, thanks!

**Reply by mathieutu** [reply]
Thanks for the release. I've updated the PR.
 
The tests for Linux [are failing](https://github.com/Homebrew/homebrew-core/actions/runs/13326711490/job/37221593306?pr=201919). Any idea what could be wrong?

**Reply by ua741** [reply]
```
==> FAILED
Full test ente-cli output
  /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/bin/bundle clean
  ==> Testing ente-cli
  ==> /home/linuxbrew/.linuxbrew/Cellar/ente-cli/0.2.3/bin/ente version
  2025/02/14 09:56:54 Please mount a volume to /cli-data/
  path does not exist: /cli-data/
  ````
It seems like that build is being run inside a Docker container. In that, we have a mandatory requirement for the mount volume `cli-data`. We can probably modify the code to bypass this check if there's no way to specify this mount directory in the build verification pipeline.

 From Readme.md
> `cli-data`` volume is mandatory, you can add more volumes for your export directory.

**Reply by mathieutu** [reply]
**Homebrew version released**!
